chapters/methodology.tex

```diff
@@ -1,6 +1,12 @@
 \section{Methodology} \label{sec:methodology}
 % detailing the approaches that were used in your experiments; this may also include implementation details in relation to the mobile architecture used
-According to \cite{sample_ref}, lorem ipsum dolor sit amet, consetetur sadipscing elitr, sed diam nonumy eirmod tempor invidunt ut labore et dolore magna aliquyam erat, sed diam voluptua. At vero eos et accusam et justo duo dolores et ea rebum. Stet clita kasd gubergren, no sea takimata sanctus est Lorem ipsum dolor sit amet. Lorem ipsum dolor sit amet, consetetur sadipscing elitr, sed diam nonumy eirmod tempor invidunt ut labore et dolore magna aliquyam erat, sed diam voluptua. At vero eos et accusam et justo duo dolores et ea rebum. Stet clita kasd gubergren, no sea takimata sanctus est Lorem ipsum dolor sit amet. Lorem ipsum dolor sit amet, consetetur sadipscing elitr, sed diam nonumy eirmod tempor invidunt ut labore et dolore magna aliquyam erat, sed diam voluptua. At vero eos et accusam et justo duo dolores et ea rebum. Stet clita kasd gubergren, no sea takimata sanctus est Lorem ipsum dolor sit amet.
+While developing the app a variety of agile development methods were applied.
+The project team used daily stand-ups to share their individual work progress and talk about current difficulties.
+A short conclusion meeting was held at the end of every day to summarize the work effort and support productive communication.
+A kanban board optimised the organisational process. Moreover, it helped the team to remember who's currently working on which task.

```

Comment:
I'd start the sentence with `The Kanban board...` to avoid having two sentences successively starting with `A` and because we only used one specific Kanban board